pokemon/src/main/java/gamarket/MoveCollection.java

```diff
@@ -0,0 +1,135 @@
+package gamarket;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
+/**
+ * Represents all the moves in the Pokemon universe. 
+ * A Pokemon will be able to hold at most 4 moves in its move-set. 
+ */
+public class MoveCollection {
+	
+	/** The current number of MOVEs in the array */
+	private int numMoves; 
+	
+	/** The array to contain all the MOVEs in the Pokemon Universe*/
+	private Move moveArray[]; 
+	
+	/**
+	 * The MoveCollection constructor.
+	 * It sets the numMoves to zero. 
+	 * It allocates a new array of Move type to 7. 
+	 * 
+	 */
+	public MoveCollection()
+	{
+		numMoves = 0; 
+		moveArray = new Move[7];
```

Comment:
new Move array should probably be set to 4 since that's the max moves a pokemon can have, unless you end up deciding to go with the move dictionary that Joe has mentioned in our meetings.

Reply:
I visioned the moveArray to hold all the moves that exists in the game.